### ISSUE TITLE
data dump for mainnet seems incorrect

### ISSUE THREAD

**Issue by aulneau** [hirosystems/chainhook#212]
the data dump that chainhooks pulls from is ~2 months old, and seems to be a different url structure than what is found https://archive.hiro.so/

```
Feb 19 23:27:39.091 INFO Downloading https://storage.googleapis.com/hirosystems-archive/mainnet/api/mainnet-blockchain-api-latest.tar.gz
```

**Comment by lgalabru** [comment]
Thanks for creating this issue.
Indeed, I noted that gap a few days ago, devops stopped producing the tsv dumps - I need to figure this out.

**Comment by aulneau** [comment]
Thanks ludo!

@CharlieC3 et all -- this would be a great help for us if there can be an update to the file that is used for chainhooks -- doesn't need to be automatic, just one bump to something more recent would be a big help at unblocking us.

**Comment by CharlieC3** [comment]
@aulneau I uploaded a fresh one just for you! It's available at the same URL as above.

Also, I didn't realize chainhooks was dependent on these TSV backups. We originally got rid of it because it provided to be redundant with the Postgres DB dump, and the way they were being created was faulty and fixing that method would take some changes we weren't prepared to make at the time.

@lgalabru Regarding something more long-term, let me think about this a bit. I may be able to start pushing out regular TSVs like we used to, but this time without the aforementioned inconsistency faults.

**Comment by aulneau** [comment]
Amazing, thank you so much @CharlieC3 ! You're the best

**Comment by lgalabru** [comment]
Thank you so much @CharlieC3 🙏

**Comment by CharlieC3** [comment]
@lgalabru All set! I was able to find a way to generate new TSVs on-demand as part of our new archival process. New TSVs will be regularly created and uploaded here:
https://archive.hiro.so/mainnet/stacks-blockchain-api/
https://archive.hiro.so/testnet/stacks-blockchain-api/

They just need to be uncompressed with `gzip` after downloading.

**Comment by lgalabru** [comment]
Re-opening this issue, it looks like it's not a 1:1 port, and changing the URL is not enough.
@CharlieC3 for some reason, my chainhook process lost its ability to uncompress these new files, did something change between the old and the new process, beside the URL?

**Comment by CharlieC3** [comment]
@lgalabru Yes, apologies for forgetting to mention this. The compressed TSV file extension is now just `.gz` instead of `tar.gz`. So you should be able to uncompress it with `gzip`, which is what `tar` uses behind the scenes anyways.
We made this change because it allowed us more flexibility in the archival process internally, but also because the TSV is a single file, and there's no need to create a `.tar` archive of a single file if our primary goal is to just compress it.

**Comment by lgalabru** [comment]
Addressed in `v0.10.0`
